src/model/handlers.py

```diff
@@ -153,4 +163,5 @@
         with open(filepath) as f:
             return web.json_response(json.load(f))
     except FileNotFoundError:
+        logger.debug(f"Request for unknown map: {modelId} / {mapId}")
```

Comment:
Are you sure you want to keep this in `debug`? I think it could be `info` or something on higher log level.

Reply:
Yes, it's a client error. The client(s) we write that access maps however should report a more serious problem if they get 404. Server-side it's simply nice to have some output so `debug` is appropriate

Reply:
agreed